### ISSUE TITLE
Accessibility report

### ISSUE THREAD

**Issue by jema28** [yalla-coop/universal-credit-support#37]
Hi @maggiehoughton @Katie-Hyde @thejoefriel,

We've gone through all the colours this afternoon and written up a report for you: https://www.notion.so/Accessibility-Report-2d505c390cb04ef0ab173c51c4330aff. It includes: 
1. An explainer of accessibility levels and scores
2. The brand colours 
3. Brand colour contrasts. As you'll see in the document there are quite colours that fail. We have a few options:
a. We tweak the colours so more reach AA(+). I have added examples of this at the bottom of the document.
b.  We avoid the failing combinations

As you'll see it's unfortunately really hard to make red and orange accessible! You have to really darken them to pass even AA+ and they lose their vibrancy. 

I've added some comparatives here that pass AA+ and also the slightly tweaked MVP colours for comparison. It's much easier to make cooler colours like blues and greens vibrant for UI.

If we want to keep the original vibrant red and orange we can do so but avoid adding white text on it and use it instead for accents (e.g. borders, lines etc)

Look forward to your thoughts on this!

**Comment by Katie-Hyde** [comment]
Hi Jem, had  a look at the report and like the suggestions you have made, in order to support branding we may have to check with our brand team but like the idea of only having set combinations to support accessibility e.g. orange / black or blue /white. Would this have a knock on effect for other organisations or would we just limit to what we agree is the highest rating ?

**Comment by maggiehoughton** [comment]
I'll raise this with Jane prior to your meeting on Monday and then maybe you can discuss possible options with her? I am worried about making a call on this without input from branding but I'll brief her about the issue beforehand.

**Comment by thejoefriel** [comment]
That's great thanks @Katie-Hyde @maggiehoughton . Agreed it makes sense so discuss with Jane.

@jema28 can you make sure to cover that in your meeting on Monday please (since I won't be there) and update this issue with then final decisions made between you and Jane on this? Thanks!

**Comment by jema28** [comment]
Hi @maggiehoughton @Katie-Hyde @thejoefriel 

Thanks for raising this Maggie. Sure thing, I will do!

**Comment by jema28** [comment]
Good morning @maggiehoughton @Katie-Hyde @thejoefriel 

Hope you enjoyed the super hot weather this weekend, it's very beautiful here! 

Wanted to share the document I'm going through with Jane this morning: https://www.notion.so/Branding-chat-528d0953e3474b4786877ccb0d7b7647. I've also recorded a loom on potential icons that we could use: https://www.loom.com/share/f66fbb7b8a364934bdcfb3cfefd25153.

Will discuss and share our meeting notes later today. Thanks!

**Comment by jema28** [comment]
Hi @maggiehoughton @Katie-Hyde @thejoefriel 

Thanks again for arranging the call with Jane, was very helpful to go through! I've added in our notes, action points and what we landed on in this doc: https://www.notion.so/Branding-chat-528d0953e3474b4786877ccb0d7b7647.

**Comment by jema28** [comment]
Hi @Katie-Hyde, to answer your question: "Would this have a knock on effect for other organisations or would we just limit to what we agree is the highest rating?" we agreed that we need at least an AA+ rating for one of the primary colours and to avoid using white on colours that fail. We'll definitely add this note in our design system and documentation when we get there so it's clear for other organisations. I can add additional guidelines on the colour choices also.

**Comment by maggiehoughton** [comment]
@jema28 Thanks for the update Jem, just read through the notes, all looking good:-)
